ui/src/Queues/UseEffectOnce.ts

```diff
@@ -0,0 +1,5 @@
+import React, {useEffect} from "react";
+
+const useEffectOnce = (fun: React.EffectCallback) => useEffect(fun, [])
```

Comment:
Co ci to daje jako osobny hook? W paru miejscach nie korzystasz z niego, ale piszesz `useEffect` z pustą tablicą depsów.

Reply:
W `QueuesTable` potrzebowałem odpalić inicjalizacji stanu która powinna wykonać się tylko raz. Jak próbowałem to zrobić przez `useEffect` z pustymi depsami to ESLint marudził, że nie przekazuje mu `queuesOverallData`. Po zdefiniowaniu `useEffectOnce` ESLint nie narzeka (zapewne jest to jakiś hack?)

Reply:
tak, po prostu oszukałeś eslinta. raczej trzeba queues overall data wpisać w depsy i zwrócić uwage na jego mutowalność

Reply:
Przeniosłem funkcję która była wywoływana wewnątrz `useEffectOnce` do hooka odpowiedzialnego za aktualizację danych. Problem zniknął, a i kod poza Hookiem się uprościł